extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -113,8 +113,13 @@ protected function renderGroup( $groupData ) {
 		$dataItems = $groupData['value'];
```

Comment:
can we rename this variable to for example 'childItems'? dataItems suggests that only <data> children of <group> are stored there